src/Proto.Cluster/Partition/PartitionIdentityActor.cs

```diff
@@ -169,26 +194,48 @@ private async Task OnActivationRequest(ActivationRequest msg, IContext context)
             //Check if exist in current partition dictionary
             if (_partitionLookup.TryGetValue(msg.ClusterIdentity, out var pid))
             {
+                if (_config.DeveloperLogging)
+                    Console.WriteLine($"Found existing activation for {msg.RequestId}");
+                
                 if (pid == null)
                 {
+                    if (_config.DeveloperLogging)
```

Comment:
Partition ID is now littered with developer logging settings to see exactly what is going on.
This should probably be replaced with instance logger. and allow instance logger to do console output